Steer/src/O2MCApplication.cxx

```diff
@@ -172,6 +174,28 @@ void O2MCApplicationBase::BeginEvent()
   mStepCounter = 0;
 }
 
+void O2MCApplicationBase::AddParticles()
+{
+  // dispatch first to function in FairRoot
+  FairMCApplication::AddParticles();
+
+  auto& param = o2::conf::SimUserDecay::Instance();
+  LOG(INFO) << "Printing \'SimUserDecay\' parameters";
+  std::cout << param << std::endl;
```

Comment:
why do you make the distinction between LOG(INFO) and std::cout here? (It should probably be the same)

Reply:
it's a mistake. it was supposed to be LOG(INFO).
I will fix that, thanks for catching it.